_posts/en/newsletters/2022-10-05-newsletter.md

```diff
@@ -0,0 +1,199 @@
+---
+title: 'Bitcoin Optech Newsletter #220'
+permalink: /en/newsletters/2022/10/05/
+name: 2022-10-05-newsletter
+slug: 2022-10-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Proposed new transaction relay policies designed for LN-penalty:**
+  Gloria Zhao [posted][] to the Bitcoin-Dev mailing list a proposal to
+  allow transactions to opt-in to a modified set of transaction relay
+  policies.  Any transaction which sets its version parameter to `3`
+  will:
```

Comment:
That's fair.  I'm going to reprase the second bullet point rather than all the rest as I think we want signal to readers that we're only summarizing the v3 rules.